docs/data/data-grid/column-definition/column-definition.md

```diff
@@ -63,13 +63,14 @@ The value generated is used for filtering, sorting, rendering, etc. unless overr
 The value formatter allows you to convert the value before displaying it.
 Common use cases include converting a JavaScript `Date` object to a date string or a `Number` into a formatted number (e.g. "1,000.50").
 
-In the following demo, a formatter is used to display the tax rate's decimal value (e.g. 0.2) as a percentage (e.g. 20%).
+In the following demo, `valueGetter` is used to convert the tax rate (e.g. `0.2`) to a decimal value (e.g. `20`),
```

Comment:
**A small suggestion**
IMO it's a really good point and usage of `valueGetter`, is essentially for the purpose of transformation, do you think it will be useful to mention other possibilities in the `valueGetter` section?

Something like: (Line 36)
```diff
- Sometimes a column might not have a corresponding value, or you might want to render a combination of different fields.
+ Sometimes a column might not have a desired value, you may use the `valueGetter` to:
+    1. Transform the value to some calculated value. (e.g. tax rate `0.2` to a decimal value `20`) 
+    2. Compute the value by combination of different fields. (e.g. `fullName` can be computed as \`${row.firstName} ${row.lastName}\`)
+    3. Deduce a value from a complex value (e.g an object containing multiple keys)
```

Reply:
Thanks, looks great now 🎉